libff/algebra/fields/fp12_2over3over2.hpp

```diff
@@ -36,6 +36,14 @@ std::istream &operator>>(std::istream &, Fp12_2over3over2_model<n, modulus> &);
 template<mp_size_t n, const bigint<n> &modulus> class Fp12_2over3over2_model
 {
 public:
+    static bigint<12 * n> euler;            // (modulus^12-1)/2
```

Comment:
This doesn't get enabled for bls12-377 (I think that's the only other curve that uses this, but I may be wrong).  Similarly for fp6_3over2.

Since it's unused on these field (i.e. we only need it on Fq and Fqe) its maybe OK to just remove it from those classes.  Otherwise we need to ensure that all curves initialize all properties of all the classes they instantiate.

This init code is really a problem with this library.  It's too easy to get the init wrong.  We should ideally have some kind of generic `Fp12_2over3over2::initialize` static function or other init function (for all field and group classes) which accepts all the appropriate parameters, or maybe some `Fp12_2over3over2_parameters` struct.  Some safer method whereby we can catch the code which does not properly set specific values.

Reply:
Created issue #74 related to this

Reply:
>  its maybe OK to just remove it from those classes.

Just to clarify: you suggest to remove the `euler` variable from the `Fp12_2over3over2_model` and `Fp6_3over2_model` classes? 

> This init code is really a problem with this library.

You refer to the init code for the curves e.g. `init_bls12_377_params` ?